### DISCUSSION TITLE
Improving config doco on wiki (disabling WorldGuard in particular)

### DISCUSSION THREAD

**Discussion by bigpresh** [GriefPrevention/GriefPrevention#333] [General]
Quick doco improvement - the wiki page on config says:

>  Note that by default, GriefPrevention will not allow a player to create a land claim which overlaps a WorldGuard region he doesn't have permission to build in. You can disable this feature in the config file.

... but doesn't tell you how to disable that.   It appears that's the `GriefPrevention.Claims.CreationRequiresWorldGuardBuildPermission` config setting.

(I'd kinda like a better config page with a full list of all config settings, and a description for each (maybe a page like the current one with an overview, then a "all config options" page that lists them all?) - if I get some time I'll try to overhaul it.)

**Comment by RoboMWM** [comment]
Yea, anyone's free to modify the wiki, my focus is largely on GP version 20 for these sorts of things.

**Comment by RoboMWM** [comment]
unsure if already addressed or not
